raiden-ts/src/constants.ts

```diff
@@ -38,3 +38,4 @@ export const RAIDEN_DEVICE_ID = 'RAIDEN';
 export const UDC_WITHDRAW_TIMEOUT = 100;
 export const DEFAULT_CONFIRMATIONS = 5;
 export const DEFAULT_MS_REWARD = parseEther('80') as UInt<32>;
+export const UDC_SECURITY_MARGIN_PERCENTAGE = 10;
```

Comment:
If this constant isn't used in SDK, please, don't add it here. SDK already has all information it needs. If you want it on the dApp, add it there

Reply:
Removed from the SDK